crates/ruff_dev/Cargo.toml

```diff
@@ -44,6 +44,11 @@ toml = { workspace = true, features = ["parse"] }
 tracing = { workspace = true }
 tracing-indicatif = { workspace = true }
 tracing-subscriber = { workspace = true, features = ["env-filter"] }
+imara-diff = "0.1.5"
+
+[features]
+# Turn off rayon for profiling
+singlethreaded = []
```

Comment:
Isn't this enabled for all builds now? I would prefer keeping as is and you can use `RAYON_NUM_THREADS=1` instead.

Reply:
No, it's not in `default` and it's also only in `format_dev`. I added this to get better flamegraphs, those with rayon looked strange.